lib/tasks/lti_key.rake

```diff
@@ -3,6 +3,8 @@ namespace :markus do
   task lti_key: :environment do
     print('Creating new private key')
     key = OpenSSL::PKey::RSA.new(2048)
+    FileUtils.mkdir_p(File.join(Settings.file_storage.lti ||
```

Comment:
It's a bit funny to recompute this when it's (presumably?) used in `LtiClient::KEY_PATH`. A better approach would be to use something like `File::dirname` to retrieve the directory path.